### PR TITLE
Multiple emails on add client

### PR DESCRIPTION
**Pull request by Shruti-Apte**
### **Notion :**
https://www.notion.so/saeloun/Frontend-Integration-Add-new-client-form-should-support-multiple-emails-a174bc5f85644ed5b0195e30028c752a?pvs=4

### **What :**
- Add new client and edit new client modals are now supporting multiple client emails.
- This PR contains backend changes from mayank's PR as well as frontend changes.
### **Why :**
- Requirement 
- It is very possible that user might want to add multiple emails when adding a new client
### **Preview :**
Screenshot 2023-05-30 at 10 10 08 AM

Screenshot 2023-05-30 at 10 26 10 AM//github.com/saeloun/miru-web/assets/72149587/9ad2496d-e8b5-4bb2-a8a9-d7259513590d">

Screenshot 2023-05-30 at 10 28 23 AM
Screenshot 2023-05-30 at 10 28 37 AM

Screenshot 2023-05-30 at 5 27 04 PM

### **Todos** (for testing):

### CODE REVIEW

File: db/migrate/20230526061908_remove_client_email.rb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+class RemoveClientEmail < ActiveRecord::Migration[7.0]
+  def change
+    # https://www.rubydoc.info/gems/strong_migrations/0.7.6
+    safety_assured { remove_column :clients, :email }
```
Comment (prasanthchaduvula) [inline db/migrate/20230526061908_remove_client_email.rb:6]: @mayankagnihotri7 let's keep this column for some days. OWe can remove this column once we have working functionality for multiple client emails on the production app

File: db/migrate/20230526062007_rename_client_emails_to_email.rb [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class RenameClientEmailsToEmail < ActiveRecord::Migration[7.0]
+  def change
+    safety_assured { rename_column :clients, :emails, :email }
```
Comment (prasanthchaduvula) [inline db/migrate/20230526062007_rename_client_emails_to_email.rb:5]: @mayankagnihotri7  No need for this migration, client emails as column name good enough

**Review by prasanthchaduvula — CHANGES_REQUESTED** [review]

File: db/migrate/20230526061821_copy_client_email_to_emails_column.rb [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+class CopyClientEmailToEmailsColumn < ActiveRecord::Migration[7.0]
+  def change
+    Client.find_each do |client|
+      client.update(emails: client.email) if client.email.present?
```
Comment (prasanthchaduvula) [inline db/migrate/20230526061821_copy_client_email_to_emails_column.rb:6]: Instead of update, use the update_column

**Review by prasanthchaduvula — CHANGES_REQUESTED** [review]

File: db/migrate/20230526061821_copy_client_email_to_emails_column.rb (hunk 3 above)
Comment (mayankagnihotri7) [reply db/migrate/20230526061821_copy_client_email_to_emails_column.rb:6]: Causes error while updating if `update_column` is used.

**Comment by apoorv1316** [comment]
@Shruti-Apte Please add system specs for it.

File: db/migrate/20230526061821_copy_client_email_to_emails_column.rb (hunk 3 above)
Comment (prasanthchaduvula) [reply db/migrate/20230526061821_copy_client_email_to_emails_column.rb:6]: what is the error

**Comment by prasanthchaduvula** [comment]
until we implement client authentication flow, this PR is on hold

**Comment by prasanthchaduvula** [comment]
Changes got implemented #1429, #1412 and #1378

**Pull request closed without merging**